src/formatting/expr.rs

```diff
@@ -2008,6 +1993,7 @@ fn choose_rhs<R: Rewrite>(
                 .indent
                 .block_indent(context.config)
                 .to_string_with_newline(context.config);
+
```

Comment:
I don't have a strong opinion one way or the other about whether there should be a newline here, though as a general comment, it's better to avoid such changes in unmodified parts of a PR in the future unless there's a really good reason for updating an unrelated section.

My guess is that this was just an accidental left over from an earlier pass at a solution that _did_ make some changes down here, so no worries!